docs/guide/areas-and-widgets/README.md

```diff
@@ -0,0 +1,147 @@
+# Areas and widgets
+
+**Areas** are core to Apostrophe's in-context editing experience. They are special fields where editors can add one or more content widgets. A **widget** is a section of structured content, such as a block of rich text, an image slideshow, or a grid of featured products. Together, these two features let editors add custom and advanced content to a website, move it around, and edit it &ndash; all within a defined modal that maintains the content design.
```

Comment:
Very good but got lost at the modal part

Reply:
Should be "model"